clients/drcachesim/reader/snappy_file_reader.cpp

```diff
@@ -110,21 +114,23 @@ snappy_reader_t::read_data_chunk(uint32_t size, chunk_type_t type)
             return false;
         }
 
-        if (!snappy::RawUncompress(&compressed_buf_[checksum_size_],
-                                   size - checksum_size_, uncompressed_buf_.data())) {
+        if (!snappy::RawUncompress(&compressed_buf_[crc_size], size - crc_size,
+                                   uncompressed_buf_.data())) {
             ERRMSG("Failed decompressing snappy chunk\n");
             return false;
         }
         src_.reset(new snappy::ByteArraySource(uncompressed_buf_.data(),
                                                uncompressed_chunk_size));
     }
 
-    size_t dummy;
-    uint32_t checksum = mask_crc32(src_->Peek(&dummy), src_->Available());
-    if (checksum != read_checksum) {
-        ERRMSG("Checksum failure on snappy block. Want %x, got %x\n", read_checksum,
-               checksum);
-        return false;
+    if (has_checksum) {
+        size_t dummy;
```

Comment:
nit: disrespectful term

Reply:
Replaced with "ignored" in this pre-existing code.